config.toml

```diff
@@ -0,0 +1,11 @@
+[boost]
```

Comment:
there is an example config, any reason to check this in as well?

Reply:
Ooops no, that's a mistake

Reply:
mind removing from the PR?

Reply:
there also have been quite some changes since this PR was opened -- you may want to start a new PR with just the changes for the `ValidatorRegistrar` on top of the latest stuff

Reply:
I will make a new PR